.claude/skills/worldarchitect-local-debugging.md

```
@@ -0,0 +1,61 @@
+# Your Project Local Debugging Reference
+
+## Log File Locations
+
+**Flask server logs:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)